src/components/TodoItem/TodoItem.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const TodoItem = ({
+  id,
+  title,
+  completed,
+  changeStatus,
+  removeTodo,
+  changeTodo,
+}) => {
+  const [isEditMode, setIsEditMode] = useState(false);
+  const [newTodoTitle, setNewTodoTitle] = useState(title);
+
+  const handleEditing = (event) => {
+    switch (event.key) {
+      case 'Enter':
+        if (newTodoTitle) {
+          changeTodo(id, newTodoTitle);
+        } else {
+          setNewTodoTitle(title);
+        }
```

Comment:
isn't it always true, I mean newTodoTitle?

Reply:
it could be an empty newTodoTitle. In this case, I don't save it